mmdeploy/codebase/mmcls/deploy/classification_model.py

```diff
@@ -84,11 +84,21 @@ def forward(self,
         cls_score = self.wrapper({self.input_name:
                                   inputs})[self.output_names[0]]
 
-        from mmcls.models.heads.cls_head import ClsHead
-        predict = ClsHead._get_predictions(
-            None, cls_score, data_samples=data_samples)
-
-        return predict
+        from mmcls.structures import ClsDataSample
+        pred_scores = cls_score
+        pred_labels = pred_scores.argmax(dim=1, keepdim=True).detach()
```

Comment:
I found that in https://github.com/open-mmlab/mmclassification/blob/5b266d9e7c49044d1fcf15d046df3c06af3ff505/mmcls/models/heads/multi_label_cls_head.py#L140, `MultiLabelClsHead` doesn't call `pred_scores.argmax`.

BTW, can we suggest mmclassification to change `predict`  , like mmdet does, so that we can reuse their `add_pred_to_datasample`
```
results_list = self.bbox_head.predict(
            x, batch_data_samples, rescale=rescale)
batch_data_samples = self.add_pred_to_datasample(
            batch_data_samples, results_list)
````